### ISSUE TITLE
Unable to start the application - Windows system error

### ISSUE THREAD

**Issue by edadams** [10se1ucgo/DisableWinTracking#257]
**INFORMATION:**

Upon downloading the application, extracting the files, right clicking the executable and selecting the option to run it as an administrator, I get an error box that says:

*The program can't start because msvcr100.dll is missing from your computer. Try reinstalling the program to fix this problem.*

[Screenshot of the error message.](http://i.imgur.com/bWvKwzH.png)

I've googled the error and it seems to happen if a system doesn't have Visual C++ Framework installed. In my Programs & Features window, I have a ton of different versions installed. [Screenshot of my Visual C++ Frameworks installed.](http://i.imgur.com/a31rZ4v.png) 

My system is completely up to date and as far as I know I have everything that I would need installed to run this. I'm probably technically able enough to do what this app does manually, but I can't work out why this won't run. 

* Windows Edition (Home/Pro/Enterprise): Pro
* Architecture (32bit/64bit): 64
* Options used: n/a

**LOG:** n/a

**Comment by Didey** [comment]
Have you tried compiling the program from source? Perhaps this could fix
the error, if not, re-download VC++ here
<https://www.microsoft.com/en-us/download/details.aspx?id=48145>.

**Comment by edadams** [comment]
That's the exact version I'd downloaded, but for the sake of testing thoroughly I removed all my previous versions, restarted, installed both of those, restarted, and the application still doesn't run. :( 

I could try running from source as a last ditch effort, yeah. I've removed a lot of the tracking by just following the readme and using [this](https://fix10.isleaked.com). The only thing I haven't yet done is added Microsoft's tracking domains to my `HOSTS` file or removed the crappy applications they install by default (but neither of those are a huge deal I guess).
